config/initializers/airbrake.rb

```diff
@@ -1,7 +1,12 @@
 # This file is overwritten on deploy
-#
+
 Airbrake.configure do |config|
+  config.api_key = ENV["ERRBIT_API_KEY"]
+  config.host    = ENV["ERRBIT_HOST"]
+  config.environment_name = ENV["ERRBIT_ENV"]
+  config.secure  = ENV["ERRBIT_API_KEY"].present?
+
   # Adding "production" to the development environments causes Airbrake not
   # to attempt to send notifications.
-  config.development_environments << "production"
+  config.development_environments << "production" unless ENV["ERRBIT_API_KEY"].present?
```

Comment:
@tadast: Do you understand why this line was there in the first place? The original commit (727c3ad9049171ba8fdcb7906c194e63cbbcaa59) doesn't offer any clues. My guess would be that it's to prevent Airbrake from sending exceptions even if the environment is set to `production` on a developer machine.

I wonder whether we should be checking for `RUNNING_ON_HEROKU` as well as `ERRBIT_API_KEY`?